tests/phpunit/unit/api/fields/includes/class-fields-test-case.php

```diff
@@ -73,6 +72,8 @@ protected function tearDown() {
 			$property = $this->get_reflective_property( $property_name );
 			$property->setValue( new \_Beans_Fields(), array() );
```

Comment:
This is no longer needed as we are resetting the Fields Container in the Test Case.